package.json

```diff
@@ -60,7 +61,8 @@
       "client/src"
     ],
     "modulePaths": [
-      "client/src"
+      "client/src",
+      "../silverstripe-admin/client/src"
```

Comment:
Don't assume silverstripe-admin is one directory up. Also include `silverstripe-admin/client/src` for installation of admin as a nested module.